src/flisp/julia_extensions.c

```diff
@@ -99,6 +99,9 @@ static int is_wc_cat_id_start(uint32_t wc, utf8proc_propval_t cat)
             (wc >= 0x2220 && wc <= 0x2222) || // ∠, ∡, ∢
             (wc >= 0x299b && wc <= 0x29af) || // ⦛, ⦜, ⦝, ⦞, ⦟, ⦠, ⦡, ⦢, ⦣, ⦤, ⦥, ⦦, ⦧, ⦨, ⦩, ⦪, ⦫, ⦬, ⦭, ⦮, ⦯
 
+	    // geometric shapes
+	    (wc >= 0x25a0 && wc <= 0x25ff) ||
+
```

Comment:
Yes, you are right. I just tried some simple function definitions without this commit and it work. I'll take it out.